source/runner/templates/githubIssueTemplate.ts

```diff
@@ -23,18 +23,33 @@ function table(name: string, emoji: string, violations: Violation[]): string {
       <th width="100%" data-danger-table="true">${name}</th>
     </tr>
   </thead>
-  <tbody>${violations
-    .map((v: Violation) => {
-      const message = isInline(v) ? `**${v.file!}#L${v.line!}** - ${v.message}` : v.message
-      return `<tr>
+  <tbody>${violations.map(violation => htmlForValidation(emoji, violation)).join("\n")}</tbody>
+</table>
+`
+}
+
+function htmlForValidation(emoji: string, violation: Violation) {
+  let message = isInline(violation)
+    ? `**${violation.file!}#L${violation.line!}** - ${violation.message}`
+    : violation.message
+
+  if (containsMarkdown(message)) {
+    message = `\n\n  ${message}\n  `
```

Comment:
This kinda keeps some weird indentation to keep the snapshots happy. Was easier to avoid breaking anything this way.